cord-vtn.json

```diff
@@ -1,26 +1,45 @@
- "org.onosproject.cordvtn" : {
+{
+    "apps" : {
+        "org.onosproject.cordvtn" : {
             "cordvtn" : {
-                "gatewayMac" : "00:00:00:00:00:01",
+                "privateGatewayMac" : "00:00:00:00:00:01",
+                "publicGateways" : [
+                    {
+                        "gatewayIp" : "10.141.192.158",
```

Comment:
I think this should be the public IP address assigned by AT&T.
I would suggest we make this "x.x.x.x" and add the note in README to remind people replacing this with real public IP